exporter/datadogexporter/README.md

```diff
@@ -1,5 +1,11 @@
 # Datadog Exporter
 
+| Status                   |                |
+| ------------------------ |----------------|
+| Stability                | [beta]         |
+| Supported pipeline types | trace, metrics |
```

Comment:
```suggestion
| Supported pipeline types | traces, metrics |
```

Reply:
Thanks for the oping Tyler, we are discussing whether the alpha or beta definition is more appropriate here for now :)